### PR TITLE
feat(CX-2788): Artist Market - UI Update (MyC Artwork Page)

### PR DESCRIPTION
**Pull request by dariakoko**
### Description

This PR resolves [CX-2788]

| iPhone full info | iPhone | iPad 9.7 full info | iPad 9.7 | iPad 12.9 full info | iPad 12.9 |
| --- | --- | --- | --- | --- | --- |
| Simulator Screen Shot - iPhone 12 mini - 2022-10-04 at 20 40 59 | Simulator Screen Shot - iPhone 12 mini - 2022-10-04 at 20 41 15 |  9 7-inch png | 9 7-inch-missing-info | 12 9-inch | 12 9-inch-missing-info |

#### Acceptance Criteria

-

#### Setup Instructions

-

### PR Checklist

- [x] I tested my changes on **iOS** / **Android**.
- [x] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [ ] I hid my changes behind a [feature flag].
- [ ] I have prefixed changes that need to be tested during a release QA with **[NEEDS EXTERNAL QA]** on the changelog.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- Artist Market - UI Update (MyC Artwork Page) - daria

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[CX-2788]: https://artsyproduct.atlassian.net/browse/CX-2788?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/Scenes/MyCollection/Screens/Artwork/Components/ArtworkInsights/MyCollectionArtworkArtistMarket.tsx [hunk 1]
```diff
@@ -31,17 +39,83 @@ export const MyCollectionArtworkArtistMarket: React.FC<MyCollectionArtworkArtist
...
+  }
+
+  const marketData: MarketDataComponents[] = []
+
+  if (!!formattedAnnualValueSold) {
+    marketData.push({
+      component: <InsightColumn name="Annual Value Sold" value={formattedAnnualValueSold} />,
+    })
+  }
+  if (!!annualLotsSold) {
```
Comment (MounirDhahri) [inline src/app/Scenes/MyCollection/Screens/Artwork/Components/ArtworkInsights/MyCollectionArtworkArtistMarket.tsx:87]: **thought:** should we maybe return the annual lots sold if they're equal to 0? maybe we can check this out with Pierluca?

Comment (MounirDhahri) [reply src/app/Scenes/MyCollection/Screens/Artwork/Components/ArtworkInsights/MyCollectionArtworkArtistMarket.tsx:87]: same for the sellThroughRate maybe

File: src/app/Scenes/MyCollection/Screens/Artwork/Components/ArtworkInsights/MyCollectionArtworkArtistMarket.tsx [hunk 2]
```diff
@@ -31,17 +39,83 @@ export const MyCollectionArtworkArtistMarket: React.FC<MyCollectionArtworkArtist
...
+          <Text variant="xs">Price Over Estimate</Text>
+          <SalePriceEstimatePerformance value={medianSaleOverEstimatePercentage} />
+        </Flex>
+      ),
+    })
+  }
+
+  if (!!formatLiquidityRank) {
+    marketData.push({
+      component: <InsightColumn name="Liquidity" value={formatLiquidityRank} />,
```
Comment (MounirDhahri) [inline src/app/Scenes/MyCollection/Screens/Artwork/Components/ArtworkInsights/MyCollectionArtworkArtistMarket.tsx:113]: **suggestion:** I think we better use `liquidityRankDisplayText` that we get from metaphysics here, we added it few weeks ago

**Review by MounirDhahri — COMMENTED** [review]
thanks for adding the screenshots on tablets as well 🙏

File: src/app/Scenes/MyCollection/Screens/Artwork/Components/ArtworkInsights/MyCollectionArtworkArtistMarket.tsx [hunk 3]
```diff
@@ -31,17 +39,83 @@ export const MyCollectionArtworkArtistMarket: React.FC<MyCollectionArtworkArtist
...
+          <Text variant="xs">{name}</Text>
+          <Text variant="lg">{value}</Text>
+        </Flex>
+      </>
+    )
+  }
+
+  const marketData: MarketDataComponents[] = []
+
+  if (!!formattedAnnualValueSold) {
```
Comment (olerichter00) [inline src/app/Scenes/MyCollection/Screens/Artwork/Components/ArtworkInsights/MyCollectionArtworkArtistMarket.tsx:82]: **nitpick:** In this case we don't necessarily need to add the `!!`. JavaScript implicitly converts the if condition to a boolean. I would argue that `formattedAnnualValueSold` is easier to read than `!!formattedAnnualValueSold`.

 ```suggestion
  if (formattedAnnualValueSold) {
```

**Review by olerichter00 — APPROVED** [review]
Looks good! Feel free to merge after addressing @MounirDhahri comments 🌟

File: src/app/Scenes/MyCollection/Screens/Artwork/Components/ArtworkInsights/MyCollectionArtworkArtistMarket.tsx (hunk 1 above)
Comment (dariakoko) [reply src/app/Scenes/MyCollection/Screens/Artwork/Components/ArtworkInsights/MyCollectionArtworkArtistMarket.tsx:87]: I have copied the logic from Force. But sure, I can bring it up on Slack

**Comment by MounirDhahri** [comment]
thanks for making the changes @dariakoko. everything LGTM now 🙏

I only have one question that is not blocking to merge this, I remember you said that the numbers are not the same between web and mobile and I am wondering if that's still the case. If so, can you make some research about it and suggest some action items to fix it. Thank you

**Review by olerichter00 — APPROVED** [review]
💯

**Comment by dariakoko** [comment]
> thanks for making the changes @dariakoko. everything LGTM now 🙏
> 
> I only have one question that is not blocking to merge this, I remember you said that the numbers are not the same between web and mobile and I am wondering if that's still the case. If so, can you make some research about it and suggest some action items to fix it. Thank you

Thank you, Mounir!
I will catch up with someone (Micayla probably) to review the numbers. Now the numbers are the same as in Force. And I receive them already formatted from the backend, so, if the change is needed, it will be done from there.

**Comment by MounirDhahri** [comment]
Cool, feel free to merge this now then. Also can you prioritize looking into the numbers asap since we can't be returning wrong stats to users.

**Pull request merged**